com.unity.robotics.urdf-importer/Runtime/Controller/Controller.cs

```diff
@@ -91,7 +85,7 @@ void Update()
         /// <param name="selectedIndex">Index of the link selected in the Articulation Chain</param>
         private void Highlight(int selectedIndex)
         {
-            if (selectedIndex == previousIndex)
+            if (selectedIndex == previousIndex || selectedIndex < 0 || selectedIndex >= articulationChain.Length) 
```

Comment:
I guess `selectedIndex < 0` and `selectedIndex >= articulationChain.Length)` won't happen due to the new method `SetSelectedJointIndex()`?

Reply:
correct. I think this check was added before the SetSelectedJointIndex(). But I think even with that, the index can get out of bounds if URDF is loaded in co routine mode while a game is running